demisto_sdk/commands/common/hook_validations/structure.py

```diff
@@ -239,3 +242,76 @@ def is_valid_file_path(self):
         if not is_valid_path:
             print_error(Errors.invalid_file_path(self.file_path))
         return is_valid_path
+
+    def print_errors(self, err):
+        """A wrapper which runs the print error message for a list of errors in yaml
+        Returns:
+            parsed error message from pykwalify
+        """
+        if ".\n" in str(err):
+            for error in str(err).split('.\n'):
+                self.print_error_msg(error)
+        else:
+            self.print_error_msg(str(err))
+
+    def print_error_msg(self, err):
+        """Returns a parsed error message from pykwalify
+        Args: an schema error message from pykwalify
+        """
+        # err is : '<SchemaError: error code 2: Schema validation failed:
+        #  - Cannot find required key \'description\'. Path: \'\''
+        step_1 = str(err).split('Path: ')
+        # step_1 is : ["<SchemaError: error code 2: Schema validation failed:\n - Cannot find required key
+        # 'description'. ", "'/script/commands/0/outputs/20'.: ", "'/'>"]
+        step_2 = step_1[1]
```

Comment:
what if step_1 has the length of 0/1?

Reply:
then an exception will be thrown and because the function print_err_msg is in try and catch then we will return the err from pykwalify